packages/react-devtools-inline/src/backend.js

```diff
@@ -61,27 +64,11 @@ function startActivation(contentWindow: window) {
   // because they are stored in localStorage within the context of the extension (on the frontend).
   // Instead it relies on the extension to pass preferences through.
   // Because we might be in a sandboxed iframe, we have to ask for them by way of postMessage().
+  // TODO WHAT HUH
```

Comment:
Whoops. Left this note to myself to follow up on this and then forgot to. Will do momentarily.